### PR TITLE
Tls script

### PR DESCRIPTION
**Pull request by davidhadas**
1. script to set config-deployment root CA + audience
2. script to unset config-deployment  root CA + audience
3. Cleanup in guard-service deployment

### CODE REVIEW

File: hack/setTLS.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Set the ROOT_CA and token audiences
+
+echo "Copy the certificate to file"
+kubectl get secret -n knative-serving knative-serving-certs -o json| jq -r '.data."ca-cert.pem"' | base64 -d >  queue-sidecar-rootca
```
Comment (psschwei) [inline hack/setTLS.sh:21]: would it be worth it use `mktemp` to create a temporary directory for the temp files used by this script?

Comment (davidhadas) [reply hack/setTLS.sh]: Done

**Review by psschwei — COMMENTED** [review]
/lgtm

probably should add an e2e test using this at some point

**Pull request merged**